modules/core/client/app/init.js

```diff
@@ -53,6 +53,7 @@
     // @link https://docs.angularjs.org/guide/production#disable-comment-and-css-class-directives
     $compileProvider.commentDirectivesEnabled(false);
     $compileProvider.cssClassDirectivesEnabled(false);
+    $compileProvider.aHrefSanitizationWhitelist(/^geo:.*/);
```

Comment:
I believe this has made every other protocol show up as "unsafe". 😁  

I'll revert this change, let's get a fix in.

Reply:
Oops, I'll check it out this week

Reply:
Fixed in https://github.com/Trustroots/trustroots/pull/846